.eslintrc.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
             files: ['*.test.js', '*.test.tsx'],
             globals: {
                 blurInput: true,
+                clearInput: true,
                 shallow: true,
                 mount: true,
                 takeModalScreenshot: true,
```

Comment:
Just realized, these should be inside storybooks typings file https://github.com/box/box-ui-elements/pull/1890/files#diff-74d358965721573d69d50f8f44457789

Reply:
You should remove it from here btw

Reply:
I got a bunch of errors after I removed the globals from this file -- seems like we need them for linting purposes. Do you know of any other solutions?

Reply:
Did you see errors in vscode or when running yarn lint?

Reply:
leave it as is i guess

Reply:
I saw errors when running yarn lint